src/Core/OrganizationFeatures/OrganizationServiceCollectionExtensions.cs

```diff
@@ -56,10 +56,13 @@ private static void AddOrganizationSponsorshipCommands(this IServiceCollection s
             if (globalSettings.SelfHosted)
             {
                 services.AddScoped<IRevokeSponsorshipCommand, SelfHostedRevokeSponsorshipCommand>();
+                services.AddScoped<ISelfHostedSyncSponsorshipsCommand, SelfHostedSyncSponsorshipsCommand>();
             }
             else
             {
                 services.AddScoped<IRevokeSponsorshipCommand, CloudRevokeSponsorshipCommand>();
+                services.AddScoped<ICloudSyncSponsorshipsCommand, CloudSyncSponsorshipsCommand>();
+                services.AddScoped<IValidateBillingSyncKeyCommand, ValidateBillingSyncKeyCommand>();
```

Comment:
These commands should be outside of the global settings `if/else`, the reason to add them to an if is if we need to _swap_ which class is loaded for the given interface, like for `IRevokeSponsorshipCommand`. In these cases, you have a unique interface for each.